### DISCUSSION TITLE
Deepgram Nova Returning Blank Transcripts

### DISCUSSION THREAD

**Discussion by ks233ever** [deepgram/community#713] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

I have production code that has been running every few errors for the last 6 months or so, never any issues. However in the past couple of days the transcript returned by Deepgram is blank suddenly, which causes many downstream issues. 

Here is the function that has been running no problem for months:

python library version is deepgram-sdk == 2.4.0     
```
from deepgram import Deepgram


api_key = os.environ['DEEPGRAM_API_KEY']
dg_client = Deepgram(api_key)
`with open(f"{filename}.mp3", 'rb') as audio:
        source = {'buffer': audio, 'mimetype': 'audio/mpeg'}
        response = dg_client.transcription.sync_prerecorded(source, {"model": "nova-2", "language": "en-US", 'detect_entities':False })
        logging.info('API response from Deepgram returned')
        logging.info(response)
        logging.info('\n')
        return response`
```

The response returns a blank transcript value of '': `{'metadata': {'transaction_key': 'deprecated', 'request_id': 'c6136713-263b-4287-8e70-fa41c82f3a94', 'sha256': '33fda37d570d2eb71d874575b2f5c6d6e76d8fc5bb621f8093bf3d6fc1b34d6e', 'created': '2024-04-18T18:16:06.181Z', 'duration': 5.9759374, 'channels': 1, 'models': ['1abfe86b-e047-4eed-858a-35e5625b41ee'], 'model_info': {'1abfe86b-e047-4eed-858a-35e5625b41ee': {'name': '2-general-nova', 'version': '2024-01-06.5664', 'arch': 'nova-2'}}}, 'results': {'channels': [{'alternatives': [{'transcript': '', 'confidence': 0.0, 'words': []}]}]}}`

When looking at the logs in the UI it shows the audio clip of 10 seconds or so, with a 200 success status that I was charged for. However the transcript is blank. What is causing this?

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

 c6136713-263b-4287-8e70-fa41c82f3a94

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
Hi @ks233ever 

Try removing the ` 'mimetype': 'audio/mpeg'` and trying again. That is no longer needed.
